content/en/docs/release-oversight/the-technical-release-team.md

```diff
@@ -0,0 +1,31 @@
+
+To assist in this detection we build tools like [sippy](https://sippy.ci.openshift.org/) and collaborate on improvements to the [release controller](https://github.com/openshift/release/tree/master/core-services/release-controller).  We investigate elusive problems in our tests and file PRs against dozens of repositories spanning as many teams.
+
+### How we do it
+For OpenShift 4.10 development our focus has been on creating a form of "quality ratchet" for OpenShift upgrades rather that only fixing tests.  Our current ratchet is implemented by a number of tests and tools that have increased the speed of regression detection in our nightly development payloads.
+* [A suite of tests that detect service disruption](https://github.com/openshift/origin/tree/master/test/extended/util/disruption): On one level these classes of tests may appear to fail randomly.  The goal of the following tools to is find the source of the disruption.
+* [The test that monitors this suite at upgrade time](https://github.com/openshift/origin/blob/0c86019e378384814e0a66157a3b7a0afd795915/test/e2e/upgrade/upgrade.go#L145): We've learned that, while the overall system is highly resilient to availability "blips", they create noise that makes understanding our CI results extremely difficult and mask regressions.  
+* [The Job Run Aggregator](https://github.com/openshift/ci-tools/tree/master/cmd/job-run-aggregator): A tool we use to run multiple instances of select jobs and aggregrate the disription data.  With the right math confidence intervals can be created to know whether or not a given payload has regressed—even in spite of test flakes.  A detailed explaination of the rationale and math behind it's implementation can be found [here](/docs/release-oversight/improving-ci-signal).
+
+### Why are regressions happening anyway?  Why not catch them pre-merge?
+We're moving that direction.  Once we catch a regression we're able to work with team on how to use the tooling mentioned above to test the fix pre-merge.
+
+As for not catching it in the first place, it's mostly a result of the size of OpenShift and all the layered offerings.  Hundreds of developers are committing many thousands of lines of code each day and it's unrelistic to run CI blocking jobs for the myriad permutations of cloud provides, layered products, deployment topologies and features.  Instead, a fully integrated payload is not created roughly every two hours that undergoes the battery of [release gating](/docs/architecture/release-gating) jobs.
```

Comment:
nit: drop "not"?  It may have originally been intended to be "now", but I don't think we've made major changes to CI/nightly release build rates recently.

Reply:
```suggestion
As for not catching it in the first place, it's mostly a result of the size of OpenShift and all the layered offerings.  Hundreds of developers are committing many thousands of lines of code each day and it's unrelistic to run CI blocking jobs for the myriad permutations of cloud provides, layered products, deployment topologies and features.  Instead, a fully integrated payload is now created roughly every two hours that undergoes the battery of [release gating](/docs/architecture/release-gating) jobs.
```

I think Trevor means this